index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
+            <p class="con-par">
+              If you have an application you are
+              interested in developing, a feature that you
+              need built or a project that needs coding.
+              I'd love to help with it.
+            </p>
+            <form action="https://formspree.io/f/xqknvevr" method="post">
+              <ul class="un-list">
+                <li class="li-item">
+                  <label for="name">
+                    <input type="text" name="name" id="name" class="name-input detail-input" placeholder="Enter your names..."/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="name">
+                    <input type="email" name="email" id="email" class="email-input detail-input" placeholder="example@gmail.com"/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="textarea">
+                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5"></textarea>
+                  </label>
+                </li>
```

Comment:
- [ ] Kindly make sure all input fields are required to be filled so a user can not submit with an empty field as required for the project. You can fix this issue by only adding the `required` attribute on each `input` and `teaxtarea` tag.